docs/advanced_topics/images/renditions.md

```diff
@@ -36,6 +36,24 @@ See also: [](image_tag)
 
 (prefetching_image_renditions)=
 
+## Generating multiple renditions for an image
```

Comment:
Suggestion: Would be nice to cross-link this on the performance page too.

Reply:
Added an anchor for this heading in [176aa23](https://github.com/wagtail/wagtail/pull/9550/commits/176aa237992beb7d2c02cc66f7ff69e517618775), but I think we'd be best to hold out on updating the performance documentation until we have some clearer ways to make use of this (e.g. a template tag)